modules/oadp-installing-oadp-rosa-sts.adoc

```diff
@@ -20,6 +20,11 @@ Restic is not supported in the OADP on ROSA with AWS STS environment. Ensure the
 
 .Procedure
 
+
+. Install the OADP Operator.
+.. In the {product-title} web console, navigate to Operators *->* OperatorHub.
```

Comment:
@HagayVider1 I see this is existing text. But, can you s/Operators *->* OperatorHub/*Operators* -> *OperatorHub*